lib/handlers/quote/schema/quote-schema.ts

```diff
@@ -47,6 +47,7 @@ export const QuoteQueryParamsJoi = Joi.object({
   forceCrossProtocol: Joi.boolean().optional(),
   forceMixedRoutes: Joi.boolean().optional(),
   protocols: Joi.stringArray().items(Joi.string().valid('v2', 'v3', 'mixed')).optional(),
+  fromAddress: Joi.string().optional(),
```

Comment:
Use the same validation as the other addresses.

Also I think we should call this `simulateFromAddress`. Its not clear from just `fromAddress` that this would cause a simulation to be run.